examples/getting_started/custom_parameters.py

```diff
@@ -168,15 +208,28 @@ def prepare_ocp(
     """
 
     ocp = prepare_ocp(
-        biorbd_model_path="pendulum.bioMod", final_time=3, n_shooting=100, min_g=-10, max_g=-6, target_g=-8
+        biorbd_model_path="pendulum.bioMod",
+        final_time=3,
+        n_shooting=100,
+        optim_gravity=True,
+        optim_mass=True,
+        min_g=np.array([-1, -1, -10]),
+        max_g=np.array([1, 1, -5]),
+        min_m=10,
+        max_m=30,
+        target_g=np.array([0, 0, -9.81]),
+        target_m=20,
     )
 
     # --- Solve the program --- #
-    sol = ocp.solve(show_online_optim=True)
+    sol = ocp.solve(show_online_optim=False)
 
     # --- Get the results --- #
-    length = sol.parameters["gravity_z"][0, 0]
-    print(length)
+    gravity = sol.parameters["gravity_xyz"]
+    mass = sol.parameters["mass"]
+    print(f"Optimized gravity: {gravity}")
+    print(f"Optimized mass: {mass}")
 
     # --- Show results --- #
+    sol.graphs()
```

Comment:
Hum.. I think it answers my question.. I like it less though because graphs is a blocking function while show_online_optim is not